src/components/Game.js

```diff
@@ -27,6 +29,21 @@ const Game = () => {
     return setStatusText(`Next player: ${xIsNext ? playerXName : playerOName}`);
   }, [history, playerX, playerO]); // eslint-disable-line react-hooks/exhaustive-deps
 
+  const showRules = () => {
+    setIsShowingGameRules(!isShowingGameRules);
+  };
+
+  const hideRules = () => {
+    setIsShowingGameRules(false);
+  };
+
+  const resetGame = () => {
+    setStepNumber(0);
+    setXIsNext(true);
+    setHistory([{ squares: Array(9).fill(null) }]);
+    setIsShowingGameRules(false);
```

Comment:
I think this one can probably be removed, too. They wouldn't be able to click reset with the game rules open, so it would already be `false`